default/scripting/techs/ship_hulls/SHP_GAL_EXPLO.focs.py

```diff
@@ -55,14 +54,22 @@
                 LocalCandidate.Population
                 + LocalCandidate.Population
                 * GROWTH_RATE_FACTOR
-                * (LocalCandidate.TargetPopulation + 1 - LocalCandidate.Population)
+                * (1 + (1 - LocalCandidate.Population) / LocalCandidate.TargetPopulation)
                 >= MIN_RECOLONIZING_SIZE
```

Comment:
what's the order of operations with `>=` vs `+` and `*` here? maybe need some brackets around `LocalCandidate` to just before `>=` ?

Reply:
https://docs.python.org/3/reference/expressions.html#operator-precedence

No, brackets not required here.